project.clj

```diff
@@ -29,6 +29,9 @@
                                     :password :env/clojars_password
                                     :sign-releases false}]]
 
+  ;; Add/remove this line to test behavior
+  ;;:dependencies [[com.kohlschutter.junixsocket/junixsocket-core "2.3.2"]]
```

Comment:
For test purposes you can add this to some of the profiles - e..g the `test` one or a dedicated `unix-sockets` profile.